test/subscription.go

```diff
@@ -132,10 +132,14 @@ func ImageFromCSV(ctx *Context, csvName, csvNamespace, imageName string) (string
 	if err != nil {
 		return "", err
 	}
-	for _, img := range csv.Spec.RelatedImages {
-		if strings.Contains(img.Name, imageName) {
-			return img.Image, nil
+	for _, d := range csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs {
+		for _, c := range d.Spec.Template.Spec.Containers {
+			for _, e := range c.Env {
+				if strings.Contains(e.Name, imageName) {
```

Comment:
To avoid weird effects, should we verify that this also contains the string `IMAGE`?

Reply:
Yeah. OK. Added that.